.gitignore

```diff
@@ -1,5 +1,6 @@
 database.db
 *pyc
+books/migrations/*
```

Comment:
I thought perhaps migrate would take care of it, from whatever version the person is upgrading from, but now that I think about it, it makes sense for makemigrations which is probably why I have those files. I was experimenting with making authors many-to-many (so it can include multiple authors, contributors, per spec). I only tried it for a few moments and then noticed likely other things needed changing as well.